.github/workflows/CI-docs.yml

```diff
@@ -0,0 +1,27 @@
+name: CI-docs
+
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+      - '!docs/code/**'
+
+jobs:
+  docs-src:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup
+        run: |
+          sudo apt install python3.9 python3-virtualenv -y
+          python3.9 -mvirtualenv sphinx
+          . sphinx/bin/activate
+          pip install -r docs/requirements.txt
```

Comment:
Any reason not to use [actions/setup-python](https://github.com/actions/setup-python)?

Reply:
No particular reason. I had just put the commands I use locally without much thought to it beyond that. I do somewhat like that this can be directly reproduced on the user's machine, if anyone is doing their own. If we were being really fancy, someone could enable the RTD to host the CI for this, which would give a preview link too. But I left that as future work, then someone can delete this file.

Reply:
It turns out Ubuntu does not have the ability to install python versions anymore, so I have switched to setup-python. Thanks for the pointer to it.